release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -584,6 +584,10 @@ This PR can be merged on release day by the Docs lead.
 ### Update the site configuration files for previous releases
 
 Update `config.toml` files for the 4 previous releases. These need to be 4 separate PRs because each release has its own `release-` branch.
```

Comment:
```suggestion
Update the `config.toml` files for the 4 previous releases. You need to open 4 separate PRs, because each release has its own `release-` branch.
```